### DISCUSSION TITLE
How do you use my containers?

### DISCUSSION THREAD

**Discussion by accetto** [accetto/ubuntu-vnc-xfce-g3#9] [Show and tell]
Hello users,

I would really appreciate, if you would describe in few words, how do you use my containers.

Any feedback is welcome, positive or negative. Both will help to improve the images.

Thanks in advance.

Regards,
accetto

**Comment by ximatar** [comment]
I use it as my Linux-based development environment on my Windows desktop.  This way, I can take advantage of Linux tooling, different dev environments and library versions, sandboxed environments, etc., without messing with my Windows setup.  Setting-up and maintaining the same capabilities using Hyper-V is too involved, cumbersome, has slower turn-around time, and overall less productive compared to using your docker images.

Thanks a lot for your effort!

**Reply by accetto** [reply]
I'm pretty late, but I want to say, that I really appreciate your feedback. Thank you. :)

**Comment by samlnx03** [comment]
I use ubuntu-vnc-xfce-g3 to teach linux basics, I run one container for each of my students, that way I can help them even in a remote way out of class.

**Reply by accetto** [reply]
Hello @samlnx03 , thank you for your feedback. This is exactly the way I've imagined it. :) 
Best wishes to you and your students.

**Comment by thesearge** [comment]
I would like to use this solution for my old iPad 2 to control smarthome dashboard for home assistant running on my NAS. Ipad 2 ist limited to iOS 9.3.5 and Safari is too old to work with Home-Assistant. Home-Assistant.app isn't available for iOS 9.3.5. Connecting via VNC to noVNC to headless browser would be a great solution. Old iPad will get again useful as dashboard station for home-assistant.

... need to get it working now ...

**Reply by accetto** [reply]
Hello,

I'm not sure if I've understood you well, but if your NAS can run Docker, then you should be able to run this container on the NAS.

However, then you'll need to manage the access from the container to the Home-Assistant application running on the NAS. Accessing the host computer from Docker containers is not so easy as it seems.

Accessing the container from the Safari browser running on the iPad 2 should work, if you manage to configure your network correctly.

However, using `noVNC` requires an HTML5 capable browser. I'm not sure about the Safari browser on iPad 2.

Regards,
accetto

**Comment by markNZed** [comment]
I'm using it as a PC in the cloud by running it on a server.

**Reply by accetto** [reply]
Thank you for your feedback. I really appreciate it. :)

**Comment by agajdosi** [comment]
Hello @accetto, I use it in [BlenderKit](https://www.blenderkit.com/) as a base image for image headless Blender container (https://github.com/BlenderKit/headless-blender-container). We provide all released minor versions of Blender and also upcoming pre-releases to Blender community. We directly then use them in automated Github actions to resize textures and optimize Blender files while keeping the same Blender version as the original .blend file had.

I could not construct image like yours from scratch, so you have really saved me and enabled me to do something nice with Blender. Thank you for a wonderfull image!

**Reply by accetto** [reply]
Hello @agajdosi,

thank you very much for you nice comment. I'm really happy to hear that the images are helpful.
I wish you success and many happy developer days.

Regards,
accetto
